airbyte-webapp/src/views/Connector/ConnectorForm/index.tsx

```diff
@@ -2,4 +2,4 @@ export * from "./types";
 export * from "./ConnectorForm";
 
 export { StartWithDestination } from "./components/StartWithDestination";
-export { FrequentlyUsedDestinations } from "./components/FrequentlyUsedDestinations";
+export { FrequentlyUsedConnectors } from "./components/FrequentlyUsedConnectors/FrequentlyUsedConnectors";
```

Comment:
I think the shorter import works, right?

```suggestion
export { FrequentlyUsedConnectors } from "./components/FrequentlyUsedConnectors";
```

Reply:
Absolutely! I forgot to update it after creating the `index.tsx` with `FrequentlyUsedConnectors` export